siti_tools/siti.py

```diff
@@ -115,13 +115,54 @@ def ti(
         return (frame_data - previous_frame_data).std()
 
 
-def eotf_1886(frame_data: np.ndarray, gamma: float = DEFAULT_GAMMA) -> np.ndarray:
-    # TODO implement
+def eotf_1886(frame_data: np.ndarray, gamma: float = DEFAULT_GAMMA, l_min: float = 0.0,
+              l_max: float = 1.0) -> np.ndarray:
+    """
+    Electro-optical transfer function defined in ITU-T Rec. BT.1886
+
+    Args:
+        frame_data: pixel values expected in the range [0, 1]
+        gamma: exponent
+        l_min: minimum luminance
+        l_max: maximum luminance
+    Note:
+        As the mapping to display luminance happens outside of this function, l_min and l_max are set to 0.0 and 1.0,
```

Comment:
If these take a different range `[0-1]`, then I think the parameters should be named differently to avoid confusion with the user-configurable `l_min`, `l_max` parameters.

Reply:
--> Change the documentation to ensure that these arguments are not changed when used in the context of the `apply_display_model` function